structs/shapes_test.go

```diff
@@ -0,0 +1,33 @@
+package structs
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPerimeter(t *testing.T) {
+	rectangle := Rectangle{10.0, 10.0}
+	got := Perimeter(rectangle)
+	want := 40.0
+	assert.Equal(t, want, got, "they should be equal")
```

Comment:
Might the message  `they should be equal?`  be redundant?

Reply:
The messages have been improved! Thx @jsalinaspolo!